teps/0007-conditions-beta.md

```diff
@@ -295,20 +295,20 @@ We can explore adding more [Operators](https://github.com/kubernetes/kubernetes/
 
 ### Skipping
 
-As it is currently in `Conditions`, when a `Guard` evaluates to `False`, the `Task` and its dependent `Tasks` will be skipped by default while the rest of the `Pipeline` will execute. However, when the `Guard` is specified to operate on a missing resource (such as `Param` or `Result`), the Pipeline will exit with a failure.
-
-To provide more flexibility when a `Guard` evaluates to `False`, we propose adding a field - `continueAfterSkip` - that:
-- is used to specify whether execute the `Tasks` that are ordering-dependent on the skipped guarded `Task`
-- defaults to `false`/`no` and users can set it to `true`/`yes` (case insensitive) to allow for execution of the rest of the branch
-- is only supported in guarded `Tasks`; there will be a validation error if `continueAfterSkip` is specified in unguarded `Tasks`
+As it is currently in `Conditions`, when `WhenExpressions` evaluate to `False`, the `Task` and its branch (of dependent `Tasks`) will be skipped by default while the rest of the `Pipeline` will execute.
 
 A `Task` branch is made up of dependent `Tasks`, where there are two types of dependencies:
-- _Resource dependency_: based on resources needed from parent `Task`, which includes Workspaces, `Results` and Resources.
-- _Ordering dependency_: based on runAfter which provides sequencing of `Tasks` when there may not be resource dependencies.
+- _Resource dependency_: based on resources needed from parent `Task`, which includes `Workspaces`, `Results` and `Resources`.
```

Comment:
kind of a bit but currently there is no way to express the workspace resource dependency - tho hopefully there will be one day https://github.com/tektoncd/pipeline/issues/3109 :D

Reply:
@bobcatfish good catch, updated it